spec/rubocop/cli_spec.rb

```diff
@@ -1627,6 +1627,25 @@ def abs(path)
       end
     end
 
+    describe '-E/--extra-details' do
+      it 'shows extra details' do
+        create_file('example1.rb', 'puts 0 # rubocop:disable NumericLiterals ')
+        create_file('.rubocop.yml',
+                    ['TrailingWhitespace:',
+                     '  Details: Trailing space is just sloppy.'])
+        file = abs('example1.rb')
+
+        expect(cli.run(['--format', 'emacs', '--extra-details',
+                        'example1.rb'])).to eq(1)
+        expect($stdout.string)
```

Comment:
Add a check that nothing is printed on `$stderr`, because currently something is.